hydro_lang/src/ir.rs

```diff
@@ -960,6 +991,10 @@ pub struct HydroIrMetadata {
     pub location_kind: LocationId,
     pub backtrace: Backtrace,
     pub output_type: Option<DebugType>,
+    /// The kind of stream/collection and its properties (ordering, retries, etc.)
```

Comment:
We no longer store the properties?

Reply:
Thank you! AI removed these, I will reintroduce.